XlsIO-Excel-Protect-UnProtect/Protect-Workbook/.NET/Protect-Workbook/ProtectWorkbook/Program.cs

```diff
@@ -18,6 +18,7 @@ static void Main(string[] args)
                 IWorkbook workbook = application.Workbooks.Open(inputStream);
                 IWorksheet worksheet = workbook.Worksheets[0];
 
+                workbook.Unprotect();
```

Comment:
Please remove unprotect. Instead, remove the protection from the input template.

Reply:
Changes done sir